### PR TITLE
(docs) date fix on 6.0.3 rel notes

### PR DESCRIPTION
**Pull request by jbondpdx**
correction to the release date

### CODE REVIEW

**Comment by justinstoller** [comment]
I assume I can merge this whenever? Did we actually hit that date?

File: documentation/release_notes.markdown [hunk 1]
```diff
@@ -12,7 +12,7 @@ canonical: "/puppetserver/latest/release_notes.html"
 
 ## Puppet Server 6.0.3
 
-Released 11 January 2019.
+Released 15 January 2019.
```
Comment (justinstoller) [inline documentation/release_notes.markdown]: Confluence says this actually shipped January 16th.

**Comment by justinstoller** [comment]
Superseded by https://github.com/puppetlabs/puppetserver/pull/1974

**Comment by tvpartytonight** [comment]
15th is actually the correct date.

**Pull request merged**